src/test/resources/features/SalesForceServiceSupport.feature

```diff
@@ -1,14 +1,27 @@
 Feature: salesforce service support demo
 
-  @salesforce
+  @salesforce_servicesupport
   Scenario Outline: salesforce service support demo
     Given Read the test data  "<TestData>" from Excel
     When navigate to Salesforce as "System Admin"
     And login to Salesforce with "System Admin" credentials
     Then click on service support button
-    Then create new account
-    Then Edit Account Details
-    Then Delete Account
+#    Then create account for service support
+#    Then Edit Account Details
+#    Then create contact for service support
+#    Then Edit Contact Details for service support
+#    Then Filter Contact Details for service support
+#    Then Remove All Filters Contact Details for service support
+#    Then Click on CasesTab for service support
```

Comment:
remove the commented line if not required

Reply:
ok